src/Microsoft.AspNetCore.Mvc.Formatters.Xml/XmlDataContractSerializerOutputFormatter.cs

```diff
@@ -132,13 +132,7 @@ protected virtual DataContractSerializer CreateSerializer(Type type)
 
             try
             {
-#if NET46
-                // Verify that type is a valid data contract by forcing the serializer to try to create a data contract
-                FormattingUtilities.XsdDataContractExporter.GetRootElementName(type);
```

Comment:
You've restored `FormattingUtilities.XsdDataContractExporter`. Should restore this too.